src/org/labkey/test/components/ui/grids/DetailDataPanel.java

```diff
@@ -121,6 +124,42 @@ public DetailTable getAliquotDetailMetaTable()
         return tables.get(1);
     }
 
+    public FileUploadField getFileField(String fieldCaption)
+    {
+        return elementCache()
+                .fileField(fieldCaption)
+                .timeout(4000)
+                .find(getComponentElement());
+    }
+
+    public String getFileName(String fieldCaption)
+    {
+        return getFileField(fieldCaption)
+                .getAttachedFileName();
+    }
+
+    public boolean isFileFieldBlank(String fieldCaption)
+    {
+        return !getFileField(fieldCaption)
+                .hasAttachedFile();
+    }
```

Comment:
Do you need this method? Won't the method above, getFileName(), return an empty string if there is no file? 
I'm not opposed to this method just curious.